client/blocks/location-search/README.md

```diff
@@ -0,0 +1,32 @@
+LocationSearch
+===
+
+A search component for searching locations via the Google Places API.
+
+## Usage
+
+```jsx
+import LocationSearch from 'blocks/location-search';
+
+export default function AwesomeLocationSearch() {
+	return (
+		<LocationSearch />
+	);
+}
+```
+
+### Props
+
+Props are displayed as a table with Name, Type, Default, and Description as headings.
+
+**Required props are marked with `*`.**
```

Comment:
I'm not sure we should keep that mention, especially since we don't have any required prop.